File: Java/program/Product.java
//Kelas Product
class Product {
    //private atribut
    private String idProduct;
    private String name;
    private String brand;
    private int price;

    //constructor
    public Product(){
    }

    public Product(String idProduct, String name, String brand, int price){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    //setter getter
    public void setIdProduct(String idProduct){
        this.idProduct = idProduct;
    }

    public String getIdProduct(){
        return this.idProduct;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getBrand() {
        return brand;
    }

    public void setBrand(String brand) {
        this.brand = brand;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

}

File: Python/program/Product.py
#Class Product

class Product:
    #Atribut
    __idProduct = ""
    __name = ""
    __brand = ""
    price = 0
    
    #constructor
    def __init__(self, idProduct, name, brand, price):
        self.__idProduct = idProduct
        self.__name = name
        self.__brand = brand
        self.price = price

    # Getter dan Setter untuk idProduct
    def get_idProduct(self):
        return self.__idProduct

    def set_idProduct(self, idProduct):
        self.__idProduct = idProduct

    # Getter dan Setter untuk name
    def get_name(self):
        return self.__name

    def set_name(self, name):
        self.__name = name

    # Getter dan Setter untuk brand
    def get_brand(self):
        return self.__brand

    def set_brand(self, brand):
        self.__brand = brand

    # Getter dan Setter untuk price
    def get_price(self):
        return self.price

    def set_price(self, price):
        self.price = price
